src/common-utils/scripts/bash_theme_snippet.sh

```diff
@@ -1,4 +1,3 @@
-
 # bash theme - partly inspired by https://github.com/ohmyzsh/ohmyzsh/blob/master/themes/robbyrussell.zsh-theme
 __bash_prompt() {
     local userpart='`export XIT=$? \
```

Comment:
Can we bump the minor version? Thanks

https://github.com/devcontainers/features/blob/38481091ba5f070d51c51053d908197926f50824/src/common-utils/devcontainer-feature.json#L3

Reply:
Changed it to 2.4.8